lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -166,10 +166,10 @@ def const_name(node)
           # rubocop:enable InternalAffairs/NodeDestructuring
           if !namespace
             [name]
-          elsif namespace.cbase_type?
-            [nil, name]
-          else
+          elsif namespace.const_type?
             [*const_name(namespace), name]
+          elsif namespace.lvar_type? || namespace.cbase_type?
```

Comment:
always make me wonder what `else` could be, but I can't think of any case

Reply:
I've tried with `@klazz::Baz` and `@@klazz::Baz`, it gets into `else` with `namespace.type` with `ivar` and `cvar`, but doesn't break the spec (no offence is detected).

Reply:
Lets :shipit: then